config/constants.config.php

```diff
@@ -12,7 +12,7 @@
 defined('APPLICATION_ENV')
     || define(
         'APPLICATION_ENV',
-        (getenv('VUFIND_ENV') ? getenv('VUFIND_ENV') : 'production')
+        (getenv('VUFIND_ENV') ? getenv('VUFIND_ENV') : 'development')
```

Comment:
Accidental commit that needs to be reverted. I recommend setting dev mode by editing /etc/apache2/conf-enabled/vufindtest.conf to uncomment the development setting, then restarting Apache. It's two steps instead of one, but it avoids this accidental commit issue, which seems to happen a lot.